app/assets/javascripts/sessions.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Similar to my comments for [this PR](https://github.com/TeresaNesteby/tackle/pull/5#discussion_r55947802), you can probably remove all the empty files generated by scaffold. We don't need it, until we actually need it. ya know?